src/snowflake/cli/plugins/sql/commands.py

```diff
@@ -10,6 +10,14 @@
 app = SnowTyper()
 
 
+def _parse_key_value(key_value_str: str):
```

Comment:
Since we are going to introduce templating across commands, not just `snow sql`, this function might end up being in some utility file so every command can access.

Reply:
Correct, I believe we have it already somewhere after adding `stage execute` I will need to rebase.

Reply:
Done